src/main/java/seedu/foodrem/model/item/itemvalidators/ItemBoughtDateValidator.java

```diff
@@ -19,24 +19,25 @@ public class ItemBoughtDateValidator implements Validator {
             String.format("The item bought date must follow the format %s.", BOUGHT_DATE_PATTERN_REGEX);
 
     // Validation for year
-    private static final int MIN_YEAR = 1000;
-    private static final String MESSAGE_FOR_YEAR_TOO_SMALL =
+    private static final int MIN_YEAR = 1900;
+    public static final String MESSAGE_FOR_YEAR_TOO_SMALL =
             String.format("The year for item bought date should be larger than %d.", MIN_YEAR);
 
-    private static final int MAX_YEAR = 2100;
-    private static final String MESSAGE_FOR_YEAR_TOO_LARGE =
+    private static final int MAX_YEAR = 2300;
+    public static final String MESSAGE_FOR_YEAR_TOO_LARGE =
             String.format("The year for ite bought date should be less than %d.", MAX_YEAR);
 
```

Comment:
Actually we should make this private and hard code it in the test case. Not expose it just to test it. Not that it would affect functionality